### ISSUE TITLE
Blend Operations: Source/Destiny inputs should be swapped

### ISSUE THREAD

**Issue by LazyEti** [AmplifyCreations/AmplifyShaderEditor-Feedback#284]
Hi, I was working with blend operations and noticed that the node's "alpha" input didn't work as expected.
After digging around a little bit I noticed that the node does work perfectly but it's simply that the Source and Destiny inputs should be swapped for it to make sense from the user's perspective.

As you can see on this example, you can expect the source input to be the original color to which the Destiny should be substracted from. However, you'll notice that the opposite result is happening:

![image](https://github.com/AmplifyCreations/AmplifyShaderEditor-Feedback/assets/39376613/2aacb486-f9fd-43e0-bcf6-f3139cf6b586)

It's easy to notice when doing a simple subtraction but it becomes much harder to wrap your head around how blend modes work when using a more complex operation.

**Comment by diogovtx** [comment]
I agree it's somewhat non-intuitive. While the original developer pretty much followed the Photoshop convention (which was my original request), I believe his selection of input port names is what contributed to the problem.

I went ahead and changed the input port names to match Photoshop convention and docs:
- Port 0 is now "Blend" (top layer)
- Port 1 is "Base" (bottom layer)

While I was at it, I fixed the formula for SoftLight for accuracy, also added a few missing Photoshop blend modes:
- DarkerColor
- LighterColor
- Hue
- Saturation
- Color
- Luminosity
